src/main/kotlin/no/nav/familie/ef/sak/opplysninger/personopplysninger/pensjon/HistoriskPensjonClient.kt

```diff
@@ -14,17 +15,29 @@ class HistoriskPensjonClient(
     private val historiskPensjonUri: URI,
     @Qualifier("azure") restOperations: RestOperations,
 ) : AbstractRestClient(restOperations, "pensjon") {
+    private val logger = LoggerFactory.getLogger(javaClass)
+
     private fun lagHarPensjonUri() =
         UriComponentsBuilder.fromUri(historiskPensjonUri).pathSegment("api/ensligForsoerger/harPensjonsdata")
             .build().toUri()
 
-    fun harPensjon(
+    val manglendeRespons = HistoriskPensjonResponse(false, "")
+
+    fun hentHistoriskPensjonStatusForIdent(
         aktivIdent: String,
         alleRelaterteFoedselsnummer: Set<String>,
-    ): HistoriskPensjonResponse {
-        return postForEntity(
-            lagHarPensjonUri(),
-            EnsligForsoergerRequest(aktivIdent, alleRelaterteFoedselsnummer),
-        )
+    ): HistoriskPensjonDto {
+        repeat(2) {
+            try {
+                return postForEntity<HistoriskPensjonResponse>(
+                    lagHarPensjonUri(),
+                    EnsligForsoergerRequest(aktivIdent, alleRelaterteFoedselsnummer),
+                ).tilDto()
+            } catch (e: Exception) {
+                logger.error("Kunne ikke kalle historisk pensjon for uthenting")
```

Comment:
Fungerer fint dette, men vi får denne erroren to ganger før vi får "UKJENT"  - jeg kan leve med dette, men hvis vi tenker det er viktig kan vi sikkert finne en løsning hvor vi logger feil en gang?

Reply:
Enig, endrer dette til å kun logge siste gangen!